mypy/suggestions.py

```diff
@@ -549,12 +550,13 @@ def find_node(self, key: str) -> tuple[str, str, FuncDef]:
         # TODO: Also return OverloadedFuncDef -- currently these are ignored.
         node: SymbolNode | None = None
         if ":" in key:
-            if key.count(":") > 1:
+            platform_key_count = 2 if sys.platform == "win32" else 1
```

Comment:
```suggestion
            # A colon might be part of a drive name on Windows (like `C:/foo/bar`)
            # and is also used as a delimiter between file path and lineno.
            # If a colon is there for any of those reasons, it must be a file+line
            # reference.
            platform_key_count = 2 if sys.platform == "win32" else 1
```

Reply:
I think that could be a helpful explainer; although I agree one could likely infer the purpose, I think being explicit and telling people in the future the purpose of this would be valuable.